tests/test_wog.py

```diff
@@ -109,6 +109,21 @@ def test_hallazeal_the_ascended():
 	assert hallazeal.dead
 
 
+def test_mark_of_yshaarj():
+	game = prepare_game()
+	mark = game.player1.give("OG_048") 
+	mark2 = game.player1.give("OG_048") 
+	wisp = game.player1.give(WISP)
+	chicken = game.player1.give("EX1_009")
+	wisp.play()
+	chicken.play()
+	assert len(game.player1.hand) == 6
```

Comment:
Whenever I need to check what's in the hand, I discard the cards from player1 at the beginning of the game (cf. other tests).